src/core/Akka/Actor/ActorBase.Lifecycle.cs

```diff
@@ -62,7 +64,7 @@ public virtual void AroundPostRestart(Exception cause, object message)
         /// <param name="message">optionally the current message the actor processed when failing, if applicable.</param>
         protected virtual void PreRestart(Exception reason, object message)
         {
-            Context.GetChildren().ToList().ForEach(c =>
```

Comment:
Don't believe the `ToList` is necessary here. Test suite will tell me if I'm wrong.